### DISCUSSION TITLE
Add keyboard shortcut configuration to Settings for the Web interface

### DISCUSSION THREAD

**Discussion by mstewart14** [immich-app/immich#6203] [Feature Request]
While there are currently keyboard shortcuts existing, the [documentation ](https://immich.app/blog/2023/07/29/update#shortcuts-for-common-actions-on-the-web) shows a very short list and the shortcuts are immutable.  As a power user I might want to set the keyboard shortcuts myself.  

- [ ] Server
- [x] Web
- [ ] Mobile

**Comment by martabal** [comment]
You can see the existing keyboard shortcuts with `?` but you can't set the keyboard shortcut yourself

**Reply by mstewart14** [reply]
thanks... just found documentation here https://immich.app/blog/2023/07/29/update#shortcuts-for-common-actions-on-the-web

**Comment by aisbergde** [comment]
I did not know, that keyboard shorcuts exists. There should be hints in the GUI. Otherwise, how a user should or could imagine about the existence of shortcuts?

**Comment by Sean-T-Moore** [comment]
One of the many things I love about immich is how well it works with the Vimium browser extension for selecting page elements using keyboard shortcuts.  However, the Vimium key to show the link hints is 'f' which conflicts with the immich 'favorite this photo' shortcut. So it would be great to be able to specify an alternative when shortcuts conflict with browser extensions.

**Comment by almogtzabari** [comment]
I think it would be nice to have a shortcut for fullscreen (for videos at least).
